operators/deploy/tenant-operator/values.yaml

```diff
@@ -12,6 +12,10 @@ configurations:
     targetClient: k8s
     user: username
     pass: password
+  nextcloud:
+    url: "https://nextcloud.crownlabs.example.com/"
+    user: username
+    pass: password
```

Comment:
These additions should also be backported in the main values.yaml file (deploy/crownlabs)